src/h5web/visualizations/heatmap/HeatmapVis.tsx

```diff
@@ -3,16 +3,22 @@ import React, { useEffect } from 'react';
 import styles from './HeatmapVis.module.css';
 import ColorBar from './ColorBar';
 import { useHeatmapStyles, useProps, useValues } from './hooks';
-import AxisGrid from './AxisGrid';
+import AxisGrid from '../shared/AxisGrid';
 import Mesh from './Mesh';
-import Tooltip from './Tooltip';
+import Tooltip from '../shared/Tooltip';
 import HeatmapProvider from './HeatmapProvider';
 import { useHeatmapConfig } from './config';
+import { Domain } from '../shared/models';
+import PanZoomMesh from '../shared/PanZoomMesh';
 
 function HeatmapVis(): JSX.Element {
   const props = useProps();
   const [mapAreaRef, heatmapStyles] = useHeatmapStyles();
 
+  const { dims, axisOffsets, data } = props;
```

Comment:
you can destructure `dims` here for more clarity: `const { [row, cols], axisOffsets, ...`